BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/PerformanceConfiguration.cs

```diff
@@ -27,6 +27,59 @@ public PerformanceConfiguration(string apiKey)
         internal float PValueCheckIntervalSeconds = 30f;
 
         //Public config
+        private const int DEFAULT_ATTRIBUTE_STRING_VALUE_LIMIT = 1024;
+        private int _attributeStringValueLimit = DEFAULT_ATTRIBUTE_STRING_VALUE_LIMIT;
+        public int AttributeStringValueLimit
+        {
+            get => _attributeStringValueLimit;

```

Comment:
As per discussions on the default endpoint and allowing the default to change in future if the value hasn't changed in the Editor, can the getter return the default rather than assigning the default value?